packages/wonder-blocks-core/components/provider-group.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from "react";
+
+type Provider<P> = {|
+    type: React.ComponentType<P>,
+    props?: P,
+|};
+
+type Props = {|
+    providers: Array<Provider<any>>,
+    children: (Array<any>) => React.Node,
+|};
+
+export default class ProviderGroup extends React.Component<Props> {
+    render() {
+        const {providers, children} = this.props;
+        const provided = [];
+        let i = 0;
+        const buildProviders = (...args) => {
+            if (i > 0) {
+                provided.push(...args);
```

Comment:
The first iteration kicks things off and isn't actually from one of the providers, so we don't want to include its args (an empty array).

Reply:
Spreading makes it impossible to support CaaF components that might pass variable numbers of args.  Not spreading makes it awkward to use.  Not sure the best direction to go in.

Reply:
I think that can be easily addressed